target/linux/mediatek/mt7622/base-files/etc/uci-defaults/05_fix-compat-version

```diff
@@ -9,7 +9,8 @@ case "$(board_name)" in
 	uci set system.@system[0].compat_version="1.1"
 	uci commit system
 	;;
-	linksys,e8450-ubi)
+	linksys,e8450-ubi|\
+	xiaomi,redmi-router-ax6s)
```

Comment:
Maybe it's a chance to remove support for this device and add a new one `redmi,ax6s` instead. I remember `Redmi` is a standalone brand. It's not a series of `Xiaomi`.

Reply:
@DragonBluep 
You are in vain trying to use marketing names.
Each Xiaomi router has a sticker on which the model name is written.
AX6S = RB03
AX3200 = RB01

Here's a look at ipq806x-based devices: https://github.com/openwrt/openwrt/tree/main/target/linux/ipq806x/files/arch/arm/boot/dts
It is not the marketing name of the router that is used, but the model name.

The marketing name must be indicated here:
`DEVICE_ALT1_MODEL :=  `

Reply:
The name should be `vendor,model`. I think we should keep Xiaomi as it's the actual vendor.

Reply:
> You are in vain trying to use marketing names.
> Each Xiaomi router has a sticker on which the model name is written.
> AX6S = RB03
> AX3200 = RB01

So the correct model should be `xxx,rb03`. Sadly, there are no Xiaomi devices using the correct model name.

> The name should be `vendor,model`. I think we should keep Xiaomi as it's the actual vendor.

At least Xiaomi devices have the `MI` logo but Redmi devices have the `Redmi` logo. Anyway, they still share the same official site. Their relationship is more like `Mercury` and `TP-Link`.

Reply:
@DragonBluep 
Certification for AX3200 (RB01): https://portal.eaeunion.org/sites/odata/_layouts/15/Portal.EEC.Registry.UI/DisplayForm.aspx?ItemId=77047&ListId=d84d16d7-2cc9-4cff-a13b-530f96889dbc
I don't see the redmi name!